### ISSUE TITLE
Could you offer the pretrained model on bsds500?

### ISSUE THREAD

**Issue by stupidman98** [xavysp/DexiNed#100]
I noticed that you said the edge on bsds500-trained model is thick.I want to see.thank you

**Comment by xavysp** [comment]
Hi, to see this difference, quantitatively, its hard to say. we need a new way, or metric for evaluation.  For a perceptual judgement, you can find  (our banner)(https://github.com/xavysp/DexiNed/blob/master/figs/DexiNed_banner.png). When the models are trained with BSDS the predictions are thick, if the model is trained with BIPED, as you can see, the predicted edges are thinner.

let me know what you think.
Xavier

**Comment by stupidman98** [comment]
In my opnion,weighted celoss may lead net to predict more edge around gt for gain less loss,because the cost from false positive is cheap.Focal loss maybe a good choice,which is briefer than CATS,and it effects in my experiments.

**Comment by xavysp** [comment]
I agree with you. However, when I started in Edge detection, I started tackling the most important problem, which is Dataset for edge detection. Once this problem is solved or at least near to be solved. We can start on optimisation, loss function, and/or deep learning (DL) architecture. At the end, the objective is to make a DL models for edge detection as simpler as the ones for image classification. 

Nice to chat with you and keep working!

Xavier
